internal/app/service_test.go

```diff
@@ -285,3 +285,49 @@ var _ = Describe("When modifying relationship of issueRepository and Service", L
 		Expect(service).NotTo(BeNil(), "service should be returned")
 	})
 })
+
+var _ = Describe("When listing serviceNames", Label("app", "ListServicesNames"), func() {
+	var (
+		db      *mocks.MockDatabase
+		heureka app.Heureka
+		filter  *entity.ServiceFilter
+		options *entity.ListOptions
+		name    string
+	)
+
+	BeforeEach(func() {
+		db = mocks.NewMockDatabase(GinkgoT())
+		options = getListOptions()
+		filter = getServiceFilter()
+		name = "f1"
+	})
+
+	When("no filters are used", func() {
+
+		BeforeEach(func() {
+			db.On("GetServiceNames", filter).Return([]string{}, nil)
+		})
+
+		It("it return the results", func() {
```

Comment:
I think the correct results in this case would imply that we have only unique names? 

I would recommend adding this to the test.